src/core/defaults.ts

```diff
@@ -22,6 +22,7 @@ const DEFAULTS: Partial<Configuration> = {
     realTimeBiasing: {
       strength: 'Medium_Increase',
       maxBiases: 25,
+      attributeDefaultBiases: 3,
```

Comment:
attributeDefaultBiases doesn't really sound related to 'max'.

Would attributeMaxBiases work better? With a note in the documentation that it can be overridden in each individual attribute's config.